docs/api/README.md

```diff
@@ -1244,7 +1244,7 @@ test('my oddOrEven saga', assert => {
     );
 
     a.equal(
-      data.gen.next().done,
+      data.gen.next().toPromise(),
```

Comment:
that's iterator's flag showing whether the iterator is finished or not. so no needs for changes

Reply:
@JSoet would u like to remove those changes here? The only needed change is [this one](https://github.com/redux-saga/redux-saga/pull/1456/files#diff-0f72f7af9b98c6cb0c52d489836df1e8R965).